tests/lua/json/stringify.lua

```diff
@@ -0,0 +1,137 @@
+local JSON = require('json')
```

Comment:
cc @Mm2PL 

what should we "recommend" / is the standard for json imports in lua.
`JSON`? `json`?

Just thinking that we should be consistent in our tests, examples, and documentation from the start to give the highest chance of plugins being consistent in their style.

Reply:
I think we should keep the name of the module and its local variable the same. 
This is [something the LuaRocks project insists on](https://github.com/luarocks/lua-style-guide#modules)

So this would be the convention:
```lua
local name = require('long.module.name')
local json = require('json')
```